gptcache/cache/factory.py

```diff
@@ -49,10 +46,18 @@ def get_ss_data_manager(scalar_store: str, vector_store: str, **kwargs):
     max_size, clean_size, dimension, top_k = _get_common_params(**kwargs)
     scalar = _get_scalar_store(scalar_store, **kwargs)
     if vector_store == "milvus":
+        if dimension <= 0:
+            raise ParamError(f"the data manager should set the 'dimension' parameter greater than zero, "
+                             f"current: {dimension}")
         vector = Milvus(dim=dimension, top_k=top_k, **kwargs)
     elif vector_store == "faiss":
+        if dimension <= 0:
+            raise ParamError(f"the data manager should set the 'dimension' parameter greater than zero, "
+                             f"current: {dimension}")        
```

Comment:
A `check_dimension` function would be better

Reply:
fixed